guides/source/active_record_encryption.md

```diff
@@ -134,15 +134,15 @@ end
 
 You might need to ignore case when querying deterministically encrypted data. There are two options that can help you here.
 
-You can use the option `:downcase`  when declaring the encrypted attribute. This will make that content is downcased before being encrypted.
+You can use the option `:downcase` when declaring the encrypted attribute. This will downcase that content before encrypting it.
 
 ```ruby
 class Person
   encrypts :email_address, deterministic: true, downcase: true
 end
 ```
 
-When using `:downcase` the original case is lost. There might be cases where you need to preserve the original case when reading the value, but you need to ignore the case when querying. For those cases you can use the option `:ignore_case` which requires you to add a new column named `original_<column_name>` to store the content with the case unchanged:
+When using `:downcase` the original case is lost. There might be situations where you need to preserve the original case when reading the value, but you need to ignore the case when querying. For those situations you can use the option `:ignore_case` which requires you to add a new column named `original_<column_name>` to store the content with the case unchanged:
```

Comment:
```suggestion
When using `:downcase`, the original case is lost. In some situations, you might want to ignore the case only when querying, while also storing the original case. For those situations, you can use the option `:ignore_case`. This requires you to add a new column named `original_<column_name>` to store the content with the case unchanged:
```

Reply:
Based on @jacobherrington's suggestion (thank you!), written as another comment for my suggestion-picking convenience.

I intentionally only made minimal changes because otherwise I could spend forever polishing texts, but fair enough :)